### PR TITLE
New widgets to handle (new) search elements

### PR DESCRIPTION
**Pull request by mganisin**
Search is shared and (more importantly) has changed. Here is update
according to new layout plus widgets to cover it.

### CODE REVIEW

File: testsuite/ui/widgets/search.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class SearchInput(TextInput):
+    """Widget representing 3scale search input used on several places"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, locator="//input[@aria-label='Search input']", **kwargs)
+
+
+class SearchButton(GenericLocatorWidget):
```
Comment (dhlavac) [inline testsuite/ui/widgets/search.py:16]: Please move the search button to buttons module and ideally Input to general widget module

**Review by dhlavac — CHANGES_REQUESTED** [review]

File: testsuite/ui/widgets/search.py (hunk 1 above)
Comment (mganisin) [reply testsuite/ui/widgets/search.py:16]: Or alternatively, search related items could be bundled together. None of these is supposed to live without the other one.

**Comment by jsmolar** [comment]
I was thinking about this for a long time. These two buttons, should be one widget: https://www.patternfly.org/v4/components/search-input

File: testsuite/ui/widgets/search.py (hunk 1 above)
Comment (mganisin) [reply testsuite/ui/widgets/search.py]: I created new SearchInput widget following model of widgetastic that couples required features of PF4 search-input

**Review by JaurbanRH — APPROVED** [review]

**Review by dhlavac — APPROVED** [review]

**Pull request merged**